Catappult/appcoins-wallet-android
# [APPC-2173] Save Phone Number

**What does this PR do?**

   Saves phone number in carrier billing, for future purchases. Use of SecureSharedPreferences.
   This PR follows the design in [APPC-2201](https://aptoide.atlassian.net/browse/APPC-2201)

**Database changed?**

   No

**Where should the reviewer start?**

- SecureCarrierBillingPreferencesRepository.kt

**How should this be manually tested?**

  Make a sucessful purchase with carrier billing. When you make a second purchase with carrier billing, the number should be saved.
  Since you may not be able to go through the whole carrier billing purchase flow, talk to me for an alternative way of testing.

**What are the relevant tickets?**

  Tickets related to this pull-request: [APPC-2173](https://aptoide.atlassian.net/browse/APPC-2173)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)
   
   Nope


**Code Review Checklist**
